docs/jumpstart/week-10.md

```diff
@@ -0,0 +1,296 @@
+file is different from our <strong> service/.../resources/properties </strong> directory as this property file is only for Heroku.
+
+Our system.properties looks like
+```ignorelang
+java.runtime.version=11
+maven.version=3.6.2
+```
+Make sure this file is in our root directory so that Heroku can easily identify this file.
+
+Once we have all of our configurations set up, we can begin deploying our backend. Navigate to our backend directory and run the command:
+
+```shell
+mvn clean install
+```
+
+This will clean our application and build it again.
+Once we verified that our application successfully built, we follow the same steps as deploying our frontend code.
+
+If we have not committed our changes to the master branch make so to do so with:
+
+```shell
+git add .
+```
+which adds all our files to our project's git repository
+
+```shell
+git commit -m
+```
+which commits our added files
+
+```shell script
+git push
+```
+which pushes our code to the master branch.
+
+
+We should already be logged into Heroku because we deployed our frontend code but if we're not, then we have to login again using:
+```shell
+heroku login
+```
+
+After logging in we run:
+
+```shell
+heroku create
+```
+which creates a new project on Heroku that represents our backend
+
+```shell
+git push heroku master
+```
+This will push our code to the Heroku server. Again, if we want to deploy a version that is not in our master branch we run the command:
+```shell
+git push heroku <branch_name>:master
+```
+This will push our designated git branch but Heroku pretends that this branch is the master branch.
+
+After pushing our code to the Heroku server we should be given a URL. This URL is now the URL that our frontend needs to make request to.
+To verify that our backend deployed successfully, we again run the command:
+```shell
+heroku open
+```
+which should open a new browser window with a message at the top saying <strong> Hello Jumpstarters! </strong>
+
+Now that our backend and frontend are deployed, we have to configure our requests URL. 
+To do this, we have to use a feature in Heroku called Heroku configuration files or Heroku secrets. Basically, this is a way for us
+to write down important information such as our backend request route in a secure and inaccessible way. To do this we 
+have to navigate back to our frontend project. Now we run the command:
+```shell
+heroku config:set REQUEST_URL=<backend_url>
+```
+
+This creates a configuration variable called REQUEST_URL which maps to the value of our backend url. Now we have to go to our React application and 
+replace where we write <strong> REACT_APP_API_DOMAIN </strong> with <strong> process.env.REQUEST_URL </strong>
+
+Therefore, everytime we make a request, our React application will look for our current processes configuration variable called REQUEST_URL, which Heroku 
+stored, and make requests using that URL.
```

Comment:
Looks good. Have you tested this out to double check though? (it might be working right now because of the .env.production file, so maybe try removing that and see if it works

Reply:
I just got it working locally, can you put something in here to tell people to remove the .env.production file if it exists for them?